src/patternfly/components/CodeSnippet/examples/CodeSnippet.md

```diff
@@ -0,0 +1,261 @@
+        {{/empty-state-body}}
+        {{#> button button--modifier="pf-m-primary"}}
+          Browse
+        {{/button}}
+        {{#> empty-state-secondary}}
+          {{#> button button--modifier="pf-m-link"}}
+            Start from scratch
+          {{/button}}
+        {{/empty-state-secondary}}
+      {{/empty-state}}
+    {{/code-snippet-code}}
+  </div>
+{{/code-snippet}}
+```
+
+### Overflow
+```hbs
+{{#> code-snippet}}
+  {{#> code-snippet-header}}
+    {{#> code-snippet-controls}}
+      {{#> input-group}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+          <i class="fas fa-copy" aria-hidden="true"></i>
+        {{/button}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
+          <i class="fas fa-download"></i>
+        {{/button}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Upload code"'}}
+          <i class="fas fa-upload"></i>
+        {{/button}}
+      {{/input-group}}
+      {{#> code-snippet-controls-overflow}}
+        {{#> overflow-menu overflow-menu--id="overflow-menu-simple"}}
+          {{#> overflow-menu-control overflow-menu-control--IsControl="true" overflow-menu-button--aria-label="Generic options"}}
+            {{#> overflow-menu-dropdown-item}}
+              Action 1
+            {{/overflow-menu-dropdown-item}}
+            {{#> overflow-menu-dropdown-item}}
+              Action 2
+            {{/overflow-menu-dropdown-item}}
+          {{/overflow-menu-control}}
+        {{/overflow-menu}}
+      {{/code-snippet-controls-overflow}}
+    {{/code-snippet-controls}}
+    {{#> code-snippet-tab}}
+      {{#> code-snippet-tab-icon}}
+        <i class="fas fa-code"></i>
+      {{/code-snippet-tab-icon}}
+    {{/code-snippet-tab}}
+  {{/code-snippet-header}}
+  {{#> code-snippet-main}}
+    {{#> code-snippet-code}}
+      code goes here 
+    {{/code-snippet-code}}
+  {{/code-snippet-main}}
+{{/code-snippet}}
+
+<br/>
+
+{{#> code-snippet}}
+  {{#> code-snippet-header}}
+    {{#> code-snippet-controls}}
+      {{#> input-group}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+          <i class="fas fa-copy" aria-hidden="true"></i>
+        {{/button}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
+          <i class="fas fa-download"></i>
+        {{/button}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Upload code"'}}
+          <i class="fas fa-upload"></i>
+        {{/button}}
+        {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Save code"'}}
+          <i class="fas fa-save"></i>
+        {{/button}}
+      {{/input-group}}
+      {{#> code-snippet-controls-overflow}}
+        {{#> overflow-menu overflow-menu--id="overflow-menu-simple"}}
```

Comment:
I'm still not sure why we need the overflow menu component here - isn't this just a dropdown?

Assuming so, since it's outside of the input group, we have a double border between the input group and overflow control. Is that intended?

<img width="248" alt="Screen Shot 2020-09-09 at 5 06 42 PM" src="https://user-images.githubusercontent.com/35148959/92659898-e224ec00-f2be-11ea-8d47-f231d12578e9.png">

Maybe instead of an input group, we just have `__controls-item` that wraps each button, then we do `__controls-item + __controls-item { margin-left: 1px; }` to fix the double border, and have a modifier of `pf-m-overflow` on the overflow item so we could position/style it differently. wdyt?